sdk/cosmos/azure-cosmos/azure/cosmos/aio/_retry_utility_async.py

```diff
@@ -194,6 +195,15 @@ async def send(self, request):
                 timeout_error.response = response
                 timeout_error.history = retry_settings['history']
                 raise
+            except ServiceRequestError as err:
```

Comment:
Are we going to subject these to DR drills of some sort?

Reply:
yep that's one of the driving factors behind the PR, filling in these gaps for DR drill